imagetest/fixtures.go

```diff
@@ -253,6 +266,16 @@ func (t *TestVM) AddAliasIPRanges(aliasIPRange, rangeName string) error {
 	return nil
 }
 
+// SetPrivateIP set IPv4 internal IP address to assign to the current test VMs.
+func (t *TestVM) SetPrivateIP(networkIP string) error {
+	if t.instance.NetworkInterfaces == nil {
+		return fmt.Errorf("Must call SetCustomNetwork prior to AddAliasIPRanges")
```

Comment:
well this function is not AddAliasIPRanges

Reply:
oops. typo.